guide/english/python/class/index.md

```diff
@@ -112,4 +112,4 @@ class Friend:
 fr1 = Friend("Nilesh","Bharti","Bihar")
 fr1.show()
 ```
-
+href="https://www.youtube.com/watch?v=ZDa-Z5JzLYM&feature=youtu.be " target="_blank"><img src="http://img.youtube.com/vi/v=ZDa-Z5JzLYM/0.jpg" alt="CLASS" width="240" height="180" border="10" /></a>
```

Comment:
This should begin with **<a** href="https.... right?

Reply:
Not only that, but we do accept PRs which only add a url.  We really want new content or improvements to the existing content.  I will close this PR.